.gitignore

```diff
@@ -0,0 +1,5 @@
+.DS_Store
+/.DS_Store
+../.DS_Store
+
+BUNDLE_PATH: ".bundle"
```

Comment:
- Line 5, please keep `.bundle` and remove `BUNDLE_PATH`: 
- To ignore .DS_Store please keep only line 1 and remove line 2 and 3. with no / in front it will match the pattern in all direcotries.
- To remove DS_store from your repository, in your local branch run `find . -name .DS_Store -print0 | xargs -0 git rm -f --ignore-unmatch` [stackoverflow link](https://stackoverflow.com/questions/107701/how-can-i-remove-ds-store-files-from-a-git-repository)